sdks/java/core/src/main/java/org/apache/beam/sdk/util/RowJson.java

```diff
@@ -159,9 +159,16 @@ public String toString() {
 
     private static final boolean SEQUENTIAL = false;
 
-    public enum MissingFieldBehavior {
-      ALLOW_MISSING,
-      REQUIRE_NULL
+    // Configure how we should attempt to parse null values when deserializing JSON
+    public enum NullBehavior {
+      // Interpret either a missing field or a null value is a null
+      ALLOW_MISSING_OR_NULL,
+      // Require nulls to be represented as a null value. If the key is missing for a nullable field
+      // an error will be thrown.
+      REQUIRE_NULL,
+      // Require nulls to be represented as a missing field. If the key exists with a null value an
```

Comment:
Maybe could rephrase the first phrase like `Requires fields with null values to be represented with a missing key.` ? - (or something else? I found it a little confusing the first time I read it) 

- Requires fields with null values to be represented with a missing key.
- Requires fields with null values to be missing from JSON representation.